pkg/session_recording/recordings.jsx

```diff
@@ -85,6 +87,32 @@
         return (ms < 0 ? '-' : '') + str;
     };
 
+    let isDateValid = function(date) {
```

Comment:
The name of the function conflicts with what it does. I would suggest renaming it to something like "parseDate", which would return the date if parsing succeeded and `null` if it didn't.

Reply:
Fixed